sharktank/tests/ops/sharded_test.py

```diff
@@ -887,6 +888,38 @@ def testReplicatedLhsAndRhs(self):
             assert ops.equal(shard, unsharded_result)
 
 
+@parameterized_class(
+    ("keepdim", "mean_dim_delta"), list(itertools.product([True, False], [-1, 0, +1]))
+)
+class MeanTest(unittest.TestCase):
+    def setUp(self):
+        self.shape = (2, 4, 6)
+        self.shard_dim = 1
+        self.mean_dim = self.shard_dim + self.mean_dim_delta
+        self.shard_count = 2
+
+    def testMeanReplicated(self):
+        tensor = torch.rand(self.shape, dtype=torch.float32)
```

Comment:
I know other tests are also guilty of this, but this will introduce nondeterminism in the test as the torch RNG was not manually seeded.